meta-client/src/main/java/org/apache/helix/metaclient/impl/zk/ZkMetaClient.java

```diff
@@ -449,12 +449,62 @@ private void cleanUpAndClose(boolean cancel, boolean close) {
     }
   }
 
+  /**
+   * MetaClient uses ZkClient to connect underlying metadata services. ZkClient current do auto
+   * reconnect infinitely. We use monitor thread in ZkMetaClient to monitor reconnect status and
```

Comment:
Thanks for working on this. This is lot of effort. 
Just curious if Apache Helix has common docs directory where we can create .md file and then refer the location of MD file here as a reference. '@see ' annotation is often used in Java code AFAIK